### ISSUE TITLE
Some feeds not imported from Google Reader opml/xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#33]
```
What steps will reproduce the problem?
1. I exported my Google Reader feeds into an opml/xml
2. I put the file into my phone and imported the feeds
3. Realized some of the feeds did not make it

What is the expected output? What do you see instead?
ALL the feeds listed in the opml/xml should be imported. Instead, I see 
only selective majority were imported.

What version of the product are you using? On what operating system?
My file version is: midp20_RSSReader-1_11_1.jar on Symbian

Please provide any additional information below.
I figured something must be up with the formatting in Google's opml. I 
opened it in Editplus (advanced notepad) and after digging for some time, 
finally noticed the pattern for those not imported:

Turns out for these, the way it was laid out in the opml is:
<outline
            text="blablabla"
...

For those that did make it, the way they were laid out in the opml is:
<outline text="blablablabla"

To test this out, I edited the ones that files by removing the carriage 
return/spaces/linefeeds or whatever crap that was in between 'outline' 
and 'text'. My files attached.

Bingo, it worked! Is there a way for you to get around this? We could ask 
Google reader to fix this bug of theirs, but I'm not sure how long that'll 
take.

Meanwhile, I'm resigned to keep checking and editing my updated opml files 
before using it in your reader.

Thanks for reading!

David
```

Original issue reported on code.google.com by `david...@gmail.com` on 12 Nov 2008 at 5:40

Attachments:
- [before_after.zip](https://storage.googleapis.com/google-code-attachments/mobile-rss-reader/issue-33/comment-0/before_after.zip)

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Sorry for the delay.  I'll look into it.  Thanks for taking the time to 
research 
this.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 22 Nov 2008 at 10:12

**Comment by GoogleCodeExporter** [comment]
```
The problem is that if the outline tag <outline
text=...
Has a carriage return or line feed after it, it does not get added to the list 
of 
feeds/bookmarks.  The workaround is to modify the OPML file to have <outline be 
on 
the same line as the rest of the tag.

This will be fixed in 1.11.2
```

Original comment by `ibunto...@gmail.com` on 8 Dec 2008 at 1:18
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I have a fix in the latest development release "Development 2".  It is posted 
on the 
main download page.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 13 Dec 2008 at 9:03

**Comment by GoogleCodeExporter** [comment]
```
Hi Irving,

Thanks, well done! I don't know if I should open a new issue topic, because 
this is
related to that Dev 2 version.

Seems to be working fine, but one peculiar thing is when I click River of News, 
my
cursor is somewhere in the middle of the whole list. Mine is a long 'river' so 
it
takes quite some time to go to the top.

Any idea what caused this?

Regards,
David
```

Original comment by `david...@gmail.com` on 20 Dec 2008 at 3:51

**Comment by GoogleCodeExporter** [comment]
```
Hi David,

Can you give some more detail?  From what you say it appears that it is only 
cosmetic.  Unless I'm mistaken, you say that while the sorting is done and the 
data 
is appended to the screen, the cursor is at the middle of the screen.  Then, 
when 
finished, the cursor goes to the top.  Then, are you able to use it without 
problems?  The only difference that I can see is that I had to add some threads 
to 
prevent hangs on some devices after I added more code.  The changes had to be 
made 
as the J2ME specification says that some methods should return very quickly.  
This 
necessitates doing more work in threads to accomplish this.  Also, making the 
font 
change does take more time as the font has to be set for every line, so it may 
be 
one of the problems with a slower phone.
I tried with my N80ie, but it doesn't do that for me, but it could be the 
number of 
items.  I could put in some progress bars to let people know that something is 
going 
on.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 21 Dec 2008 at 11:52

**Comment by GoogleCodeExporter** [comment]
```
Hi Irving,

After re-reading, I realize why you misunderstood me. Sorry for the non-clarity.

Actually, after finished sorting, my cursor is still in the middle. Ideally it 
should
be at the top right? What I meant was that I could go up using my arrow buttons 
to
reach the top (i think the previous version I did that because it was just a 10 
or so
lines from the top), but this time it's right in the middle making it too far 
to be
considered a good workaround for me.

Regarding river of news speed, even though i have a long list, it's still quite 
fast
for me. Only the above issue. So, does your phone cursor end up in the middle 
or at
the top?

Thanks.
David
```

Original comment by `david...@gmail.com` on 21 Dec 2008 at 2:40

**Comment by GoogleCodeExporter** [comment]
```
Hi David,

It does not happen to me.  I can do an easy coding change to workaround it I 
think.  
I can set the selected element in the list (screen).  If that doesn't work for 
you, 
I can try something else that you can try.  I'm working on dev-3 which I will 
put in 
a change to help you.  If that doesn't work, I can try again in dev-4.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 22 Dec 2008 at 5:00
- Added labels: **OpSys-All**

**Comment by GoogleCodeExporter** [comment]
```
Hi David,

I have posted development 3 release which hopefully will fix your problem.  Let 
me 
know if this helps.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 23 Dec 2008 at 4:27

**Comment by GoogleCodeExporter** [comment]
```
Hi Irving,

Dev 3 is working great, problem solved for the cursor in river of news to start 
at
the top.

The font and wrapping plus square brackets look good too. Thanks!

David
```

Original comment by `david...@gmail.com` on 23 Dec 2008 at 3:46

**Comment by GoogleCodeExporter** [comment]
```
Fixed on Beta 1 Version 1.11.2

```

Original comment by `ibunto...@gmail.com` on 16 Feb 2009 at 2:38

**Comment by GoogleCodeExporter** [comment]
```
This is fixed on version 1.11.2.
```

Original comment by `ibunto...@gmail.com` on 14 May 2009 at 5:11
- Changed state: **Fixed**
- Added labels: **Milestone-Release1.11.2**
